macros/sql/pivot.sql

```diff
@@ -44,13 +45,15 @@ Arguments:
                alias=True,
                agg='sum',
                cmp='=',
+               distinct=False,
```

Comment:
I think new arguments should go at the bottom of the list, incase someone (silly) is relying on the order of arguments for optional args 😓

Reply:
Of course. Now amended!